src/main/java/dev/t3d/happraisal/controller/InterviewController.java

```diff
@@ -0,0 +1,35 @@
+package dev.t3d.happraisal.controller;
+
+import dev.t3d.happraisal.service.InterviewService;
+import dev.t3d.happraisal.entity.Interview;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.websocket.server.PathParam;
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/interviews")
+@RequiredArgsConstructor
+public class InterviewController {
+
+
+    private final InterviewService interviewService;
+
+    @GetMapping
+    public ResponseEntity<List<Interview>> findAll() {
```

Comment:
This method gives org.springframework.dao.DataIntegrityViolationException: could not execute statement; SQL